assets/js/components/utils/index.js

```diff
@@ -43,3 +44,38 @@ export const getProducts = ( { selected = [], search } ) => {
 		return uniqBy( flatten( data ), 'id' );
 	} );
 };
+
+const getProductTagsRequests = ( { selected = [], search } ) => {
+	const requests = [
+		addQueryArgs( '/wc/v3/products/tags', {
+			per_page: limitTags ? 100 : -1,
+			orderby: limitTags ? 'count' : 'name',
```

Comment:
Out of curiosity, why are we changing the _orderby_ depending on this? Was that in the specs or is there another reason for that?

Reply:
I did this so that if we're only showing a handful of tags they are name sorted (so you can find the tag you want easily or search), but if you have pages and pages of tags you get the tags with the most products first, not empty or less-used tags.  Basically, most used first.